cypress/e2e/default/dynamic-routes.cy.ts

```diff
@@ -61,12 +61,13 @@ describe('Dynamic Routing', () => {
       expect(res.body).to.contain('Under the Dome')
     })
   })
-  it('renders fallback page via ODB on a non-prerendered dynamic route with fallback: true', () => {
+  it('does not render fallback page via ODB on a non-prerendered dynamic route with fallback: true', () => {
+    // unfortunately there is a problem with `fallback: true` in ODB context - the fallback would be cached indefinitely
+    // so visits to those pages would always render the fallback first and browser client would later re-render with correct
+    // content. As this is not ideal the `fallback: true` is treated as `fallback: blocking`.
     cy.request({ url: '/getStaticProps/withFallback/3/', headers: { 'x-nf-debug-logging': '1' } }).then((res) => {
       expect(res.status).to.eq(200)
-      // expect 'odb' until https://github.com/netlify/pillar-runtime/issues/438 is fixed
       expect(res.headers).to.have.property('x-nf-render-mode', 'odb')
-      // expect 'Bitten' until the above is fixed and we can test for fallback 'Loading...' message
       expect(res.body).to.contain('Bitten')
     })
```

Comment:
This is actually not changing any assertions, just update the comment why the assertions will stay as they are. Visiting https://651d4678cf52d90008b40118--netlify-plugin-nextjs-demo.netlify.app/getStaticProps/withRevalidate/withFallback/3/ (older preview) shows the problematic behavior (that fallback that flashes before browser client update) - you can also just check with JS disabled in browser and then you never get to see actual proper content